dotnet/src/SemanticKernel.Abstractions/AI/FunctionChoiceBehaviors/RequiredFunctionChoiceBehavior.cs

```diff
@@ -19,15 +21,23 @@ internal sealed class RequiredFunctionChoiceBehavior : FunctionChoiceBehavior
     /// <summary>
     /// Indicates whether the functions should be automatically invoked by AI connectors.
     /// </summary>
-    private readonly bool _autoInvoke;
+    private readonly bool _autoInvoke = true;
 
     /// <summary>
     /// Functions selector to customize functions selection logic.
     /// </summary>
     private readonly Func<FunctionChoiceBehaviorFunctionsSelectorContext, IReadOnlyList<KernelFunction>?>? _functionsSelector;
 
     /// <summary>
-    /// Initializes a new instance of the <see cref="AutoFunctionChoiceBehavior"/> class.
+    /// Initializes a new instance of the <see cref="RequiredFunctionChoiceBehavior"/> class.
+    /// </summary>
+    [JsonConstructor]
+    public RequiredFunctionChoiceBehavior()
+    {
+    }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="RequiredFunctionChoiceBehavior"/> class.
     /// </summary>
     /// <param name="functions">
     /// Functions to provide to AI model. If null, all <see cref="Kernel"/>'s plugins' functions are provided to the model.
```

Comment:
I wanted to do it in one of the 'improvement' follow-up PRs to simplify the code review for this one, but since the review is already done and this was explicitly called out, I've updated the term in this PR.